lib/cartopy/tests/test_crs.py

```diff
@@ -175,9 +175,11 @@ def test_transform_points_xyz(self):
     def test_globe(self):
```

Comment:
Interesting that this test hasn't changed based on the fact that the globe changed. Shows that the test is of questionable value. Perhaps the first thing to fix is the name of the variables (``rugby_globe`` -> ``footy_globe`` and ``footy_globe`` -> ``tennisball_globe``). And then we should think about whether this test needs to remain at all... 

Thoughts?

Reply:
I mean, that's kind of the point; Mollweide doesn't support elliptical globes, so specifying `ellipse` or `semiminor_axis` does nothing.

This test doesn't actually check that the results are different in the end...

Reply:
Oh, perhaps I was a bit unclear; that's the point of the changes in this file. As to the point of the test, I think it's just testing that the `globe` does _something_, though maybe it's become a bit redundant by the individual `test_eccentric_ellipse` tests in the `crs` directory.

Maybe we should add a `test_globe` for the ones that only support spheres and remove this test?

Reply:
Yep. That works.

Reply:
> Maybe we should add a test_globe for the ones that only support spheres and remove this test?

Either that, or we keep the test but fix the variable name. Either works for me. I'll defer this though in order to move the PR forwards.